### PR TITLE
feat: rax-gesture-view support ts

### PR DESCRIPTION
**Pull request by recover758126**

### CODE REVIEW

File: packages/rax-gesture-view/demo/index.jsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 /** @jsx createElement */
-import {createElement, Component, render} from 'rax';
+import { createElement, Component, render } from 'rax';
 import GestureView from '../src/index';
 import View from 'rax-view';
-import { isWeex } from 'universal-env';
-import * as DriverDOM from 'driver-dom';
-import * as DriverWeex from 'driver-weex';
+import Text from 'rax-text';
+import DU from 'driver-universal';
```
Comment (SoloJiang) [inline packages/rax-gesture-view/demo/index.jsx]: `DriverUniversal`

Comment (recover758126) [reply packages/rax-gesture-view/demo/index.jsx]: renamed

File: packages/rax-gesture-view/package.json [hunk 2]
```diff
@@ -38,23 +38,22 @@
     "universal-env": "^3.0.0"
   },
   "peerDependencies": {
-    "rax": "^1.0.0"
+    "rax": "^1.0.0",
+    "@rax-types/rax": "^1.0.6"
```
Comment (yacheng) [inline packages/rax-gesture-view/package.json]: 建议替换为 rax-types

Comment (recover758126) [reply packages/rax-gesture-view/package.json]: 已经替换

**Pull request merged**